packages/web/README.md

```diff
@@ -2,279 +2,60 @@
 
 ![npm](https://img.shields.io/npm/v/@lottiefiles/dotlottie-web)
 ![npm bundle size](https://img.shields.io/bundlephobia/minzip/@lottiefiles/dotlottie-web)
-![npm](https://img.shields.io/npm/dm/@lottiefiles/dotlottie-web)
+![npm](https://img.shields.io/npm/dw/@lottiefiles/dotlottie-web)
```

Comment:
@theashraf This changes the nb of downloads by a lot, do you know what the difference is?

Reply:
@samuelOsborne 
dm -> monthly downloads
dw -> weekly downloads